app/controllers/team_incomes_controller.rb

```diff
@@ -0,0 +1,24 @@
+class TeamIncomesController < ApplicationController
```

Comment:
I don't love the naming inconsistency here. I'm not exactly sure what to call this. On the client stuff I called it "public_relations" but in here I called it "team_incomes". It's kind of the overall mobile dashboard, so maybe that's a better name? It's kind of Income + PR + News + Terror Tracker all in one.